_includes/map.html

```diff
@@ -25,9 +25,9 @@
   </div>
 </div>
 
-<!-- Make compatibility with Firefox 36-44 -->
+<!-- Start of ECMA 5 -->
```

Comment:
This comment is not needed as it doesn't provide any information on the functionality.

Reply:
Done.